### PR TITLE
BaSP-TG-19: create delete and put tests for tasks entity

### PR DESCRIPTION
**Pull request by JuliBalcaza**

### CODE REVIEW

**Review by kolodani — COMMENTED** [review]
great work, you can add test of errors too

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "app_name is a system that provides Human Resources services to companies.",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
+    "test": "jest --verbose",
```
Comment (lmalvasia) [inline package.json:7]: Why did you add this?

File: src/tests/tasks.test.js [hunk 2]
```diff
@@ -1,5 +1,66 @@
...
+  describe('PUT /tasks', () => {
+    const mockTask = {
+      description: 'Hello world',
+    };
+    test('should return status code 200', async () => {
+      const response = await request(app).put('/tasks/63548d1929a3478b830780cb').send(mockTask);
+      expect(response.status).toBe(200);
+      expect(response.body.error).toBeFalsy();
+      expect(response.body.error).toBeDefined();
+      expect(response.body.message).toBe(`${response.body.message}`);
```
Comment (lmalvasia) [inline src/tests/tasks.test.js:53]: This is equal to have `expect(true).toBe(true)`. It's wrong.
You should have something like:
```suggestion
      expect(response.body.message).toEqual('Task already updated');
```

File: src/tests/tasks.test.js [hunk 3]
```diff
@@ -1,5 +1,66 @@
...
+    });
+  });
+
+  describe('DELETE /:id', () => {
+    test('should return status code 200', async () => {
+      const response = await request(app).delete('/tasks/63548d0f2e375d491a94b171').send();
+      expect(response.status).toBe(200);
+      expect(response.body.error).toBeFalsy();
+      expect(response.body.data).toBeDefined();
+      expect(response.body.message).toBe(`${response.body.message}`);
```
Comment (lmalvasia) [inline src/tests/tasks.test.js:63]: This is wrong.

File: src/controllers/tasks.js [hunk 4]
```diff
@@ -51,7 +51,7 @@ export const createTask = async (req, res) => {
     });
     const resultTask = await newTask.save();
     return res.status(201).json({
-      message: 'Task create succesfully',
+      message: 'Task create successfully',
```
Comment (leimeter-joaquin) [inline src/controllers/tasks.js:54]: created

**Review by leimeter-joaquin — COMMENTED** [review]
It's nice that you fixed these mistaked in the controller :)

File: package.json (hunk 1 above)
Comment (JuliBalcaza) [reply package.json:7]: sorry, we added it just so that the 'describe' of the test functions can be seen by the console, because otherwise it was not visible. I will remove it.

Comment (JuliBalcaza) [reply package.json:7]: AAAAANNNNDDDD.... I forgot to remove it when I did git add

**Review by leimeter-joaquin — APPROVED** [review]
